### PR TITLE
Reminder and HowTo for enabling customer experience improvement program ...

### PR DESCRIPTION
**Pull request by tommykdev**
- created ExperienceImprovementEnable.md / htm
- adjusted WhatsNew.md

### CODE REVIEW

**Comment by ghost** [comment]
Hi **@tommykdev**, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

File: ExperienceImprovementEnable.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: code
+title: Help improve program
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: Can we change the title to "Help improve the Windows Developer Program for IoT"?

File: ExperienceImprovementEnable.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+permalink: /ExperienceImprovementEnable.htm
+---
+
+### Helping us, helps you
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: 1 hash

File: ExperienceImprovementEnable.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+We constantly endeavor to refine the developer program, and encourage you to opt-in to our Customer Experience Improvement Program by following the steps below so that we can understand how the program can be improved. Turning this on enables us to make a better product that you clearly use - Helping us, helps you.
+
+### At a high level
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: 2 hash

File: ExperienceImprovementEnable.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+In order for us to get product insights, you'll need to associate your developer session with the Windows-On-Devices Development Program.  This requires using the GalileoWatcher tool to help with this association, along with enabling Visual Studio telemetry.
+* GalileoWatcher:  Run this tool to find your connected development board.
+* Visual Studio:  Opt-in to Experience Improvement.
+
+## How to enable Customer Experience Improvement ...
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: 1 hash

File: ExperienceImprovementEnable.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+### 1) Select your connected board from GalileoWatcher.
+  * Make sure your Galileo board is connected to the same subnet as your developer PC, or to the PC directly.
+  * Start GalileoWatcher.  It is the locator tool which was installed on your PC from the WindowsDeveloperProgramforIOT.MSI, and used to enumerate board addresses.
+  * Select your board name that shows up in GalileoWatcher.  You can optionally right-click to perform one of the listed actions.  By seeing and selecting your board, this associates your device with the Visual Studio telemetry pipeline, as part of the program.
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: Is clicking the galileo really required?

File: ExperienceImprovementEnable.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  * If your firewall settings were missed, or you do not see your board listed, review the GalileoWatcher troubleshooting section at [TroubleShooting](http://ms-iot.github.io/content/TroubleShooting.htm)
+
+### 2) Opt into Visual Studio Experience Improvement.  As with any pre-release evaluation, many improvements result from this feedback process.
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: If this is a title, it should be short. When viewed online the title is a little overwhelming. (I'd move the second sentence down)

File: ExperienceImprovementEnable.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  * Launch Visual Studio as an administrator
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: Pictures would really help

File: ExperienceImprovementEnable.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  * Select HELP from the menu bar, and click "Customer Feedback Options".
+  * In the "Visual Studio Experience Improvement Program" dialog, make sure the radio button for "Yes, I am willing to participate" is selected, and click OK.
+  * Whenever Visual Studio and your machines are re-started, session telemetry may be forwarded.
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: I don't understand what this means

File: ExperienceImprovementEnable.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+### At a high level
+In order for us to get product insights, you'll need to associate your developer session with the Windows-On-Devices Development Program.  This requires using the GalileoWatcher tool to help with this association, along with enabling Visual Studio telemetry.
+* GalileoWatcher:  Run this tool to find your connected development board.
+* Visual Studio:  Opt-in to Experience Improvement.
+
+## How to enable Customer Experience Improvement ...
+
+### 1) Select your connected board from GalileoWatcher.
+  * Make sure your Galileo board is connected to the same subnet as your developer PC, or to the PC directly.
+  * Start GalileoWatcher.  It is the locator tool which was installed on your PC from the WindowsDeveloperProgramforIOT.MSI, and used to enumerate board addresses.
```
Comment (ooeygui) [inline ExperienceImprovementEnable.md]: Isn't galileo watcher started by default?

**Comment by tommykdev** [comment]
I committed and pushed up all of those layout suggestions
- main title filled out for the program name
- adjusted heading sizes (hash and length suggestions)
- removed confusing step (the part about restarted), since it should occur naturally.
- added pics

**Pull request merged**